.secrets.baseline

```diff
@@ -0,0 +1,129 @@
+{
+  "exclude": {
+    "files": "vendor/*|go.sum|^.secrets.baseline$",
+    "lines": null
+  },
+  "generated_at": "2021-02-19T15:56:21Z",
+  "plugins_used": [
+    {
+      "name": "AWSKeyDetector"
+    },
+    {
+      "name": "ArtifactoryDetector"
+    },
+    {
+      "base64_limit": 4.5,
+      "name": "Base64HighEntropyString"
+    },
+    {
+      "name": "BasicAuthDetector"
+    },
+    {
+      "name": "BoxDetector"
+    },
+    {
+      "name": "CloudantDetector"
+    },
+    {
+      "name": "GheDetector"
+    },
+    {
+      "hex_limit": 3,
+      "name": "HexHighEntropyString"
+    },
+    {
+      "name": "IbmCloudIamDetector"
+    },
+    {
+      "name": "IbmCosHmacDetector"
+    },
+    {
+      "name": "JwtTokenDetector"
+    },
+    {
+      "keyword_exclude": null,
+      "name": "KeywordDetector"
+    },
+    {
+      "name": "MailchimpDetector"
+    },
+    {
+      "name": "PrivateKeyDetector"
+    },
+    {
+      "name": "SlackDetector"
+    },
+    {
+      "name": "SoftlayerDetector"
+    },
+    {
+      "name": "StripeDetector"
+    },
+    {
+      "name": "TwilioKeyDetector"
+    }
+  ],
+  "results": {
+    "assets/oauth-openshift/oauth-browser-client.yaml": [
+      {
+        "hashed_secret": "d2e2ab0f407e4ee3cf2ab87d61c31b25a74085e5",
+        "is_secret": false,
+        "is_verified": false,
+        "line_number": 14,
```

Comment:
Why are we placing a secret in a configmap and not in a Kubernetes secret?  I know this comment is unrelated to this PR but nonetheless, it is a security problem.

Reply:
The configmap is used to deliver an arbitrary CR to the child cluster via the boostrap pod. I'll open an issue to switch to a secret and have the bootstrap pod apply both configmaps and secrets.